coverage_files/input_file

```diff
@@ -1,4 +1,7 @@
-coverage_files\test*.simple
+
+
 [jacoco]
+coverage_files/test*.simple
+
 [simple]
-coverage_files\coverage.simple
\ No newline at end of file
+coverage_files/coverage.simple
```

Comment:
why did you change this to a forward slash? can we safely handle windows paths? Windows users probably expect backslashes to work

Reply:
This failed on the build server for Linux: https://travis-ci.com/github/cqse/teamscale-upload/jobs/396920590

As Windows seems to support both slashes as path separators, I would suggest we normalize all slashes to forward slashes. This way, it should work for both.

Another solution would be